common-lib/src/cmr/common/util.clj

```diff
@@ -932,3 +932,12 @@
           #(string/replace % "_" "-")
           #(string/replace % #"^CMR_" ""))
     #(string/starts-with? %1 "CMR_")))
+
+(defn get-index-or-nil
+ "Get the index of the key in the string. Return nil if the key does not
```

Comment:
I think of a key in a map when I see `key`. It looks like this should be called `substring` instead.